src/spectrum.js

```diff
@@ -79,10 +79,10 @@
         return contains(style.backgroundColor, 'rgba') || contains(style.backgroundColor, 'hsla');
     })(),
     replaceInput = [
-        "<div class='sp-replacer'>",
+        "<button type='button' class='sp-replacer'>",
             "<div class='sp-preview'><div class='sp-preview-inner'></div></div>",
             "<div class='sp-dd'>&#9660;</div>",
-        "</div>"
+        "</button>"
```

Comment:
Hi !
I don't think div inside a button is valid html :)

Reply:
I don't recall if it is valid according to HTML5 specs, but otherwise `<span>` is .. or the `<div>` can use `role="button"`.